sigridci/sigridci.py

```diff
@@ -156,14 +154,24 @@ def getRequestUploadPath(self, systemExists):
         return path
         
     def uploadBinaryFile(self, url, upload):
+        for attempt in range(self.RETRY_ATTEMPTS):
+            try:
+                return self.attemptUpload(url, upload)
+            except urllib.error.HTTPError as e:
+                log("Retrying upload")
+                time.sleep(self.POLL_INTERVAL)
+        
+        log("Sigrid is currently unavailable")
```

Comment:
Do you really want this to be `"Sigrid is currently unavailable"` and not the original `"Uploading file failed"`, or even better probably `f'Uploading file failed after {self.RETRY_ATTEMPTS} attempts'`? (As this is also/mainly added for the case where Sigrid is available but the upload times-out for some other reason.)

Reply:
Good one, will change.